java/core/src/main/java/com/lancedb/lance/DatasetFragment.java

```diff
@@ -11,7 +11,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
```

Comment:
Will these checks to be enforced in CI in the future?

Reply:
Yes, if this PR is merged, it would be enforced in the CI.